modules/log6x-quickstart-viaq.adoc

```diff
@@ -0,0 +1,149 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/logging-6.0/log6x-about.adoc
```

Comment:
Why are we duplicating already duplicated content yet again over a single field value?

Reply:
https://issues.redhat.com/browse/OBSDOCS-1494